src/sections/Landscape/LandscapeGrid.style.js

```diff
@@ -149,7 +149,7 @@ export const LandscapePageWrapper = styled.div`
 		margin: 2.5em 0;
 	}
 	.vertical-timeline::before {
-		background: #1E2117;
+		background: #fff;
```

Comment:
Please check the code on how we are switching colors based on the `theme` value, and update the code accordingly.
Use the `primaryColor` for the dark theme and `tertiaryColor` for the light theme

Reply:
Yes follow that what @Nikhil-Ladha said (it's much better to follow). 

Please don't change the colour of that centre bubble. It looks good on light and dark background.

Reply:
Use the above mentioned colors, please.